torchbiggraph/converters/import_from_tsv.py

```diff
@@ -40,26 +45,18 @@ def collect_relation_types(
     relation_configs: List[RelationSchema],
     edge_paths: List[Path],
     dynamic_relations: bool,
-    rel_col: Optional[int],
+    edgelist_reader: EdgelistReader,
     relation_type_min_count: int,
 ) -> Dictionary:
 
     if dynamic_relations:
-        if rel_col is None:
+        if edgelist_reader.rel_col is None:
```

Comment:
Having the `rel_col` attribute is not part of the EdgelistReader interface. It seems we could just raise the same error if any of the `rel_word`s returned below is None.